packages/theme/components/ProductDetails.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
to albo mozemy te 2 poprzednie komponenty tutaj zamknac i wyifowac (mniejszy cognitive overload z lataniem po plikach) albo wyifowac je w product page